README.md

```diff
@@ -77,7 +77,7 @@ let validationResult = verifiedIdRequirement.validate()
 
 Once all of the requirements are fulfilled, you can double check that the request has been satisfied by calling the `isSatisfied` method on the request object. 
 ```Swift
-let isSatisfied = await presentationRequest.isSatisfied()
```

Comment:
await should be removed from all external interface calls in README, right?

Reply:
This is a typo. isSatisfied should not have await but the rest should when some async process happens like network calls.